config/locales/en.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright © Mapotempo, 2016
+# Copyright © Mapotempo, 2020
```

Comment:
Since file was empty I assumed I should change year but I guess we can keep 2016.

Reply:
Yeah file creation date is the important one because it demonstrates that "you already had this in mind". To be honest, I have no knowledge about copyrights but I know this one because I made a similar change which was rejected due to this rule :)